### PR TITLE
upload_dataframe method

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Feature-Upload im Python-Paket](https://trello.com/c/zLLlJEUp/238-feature-upload-im-python-paket)

File: geoengine/datasets.py [hunk 1]
```diff
@@ -0,0 +1,328 @@
+from __future__ import annotations
+from enum import Enum
+from attr import dataclass
+from geoengine.types import TimeStep, TimeStepGranularity, VectorDataType
+from geoengine.auth import get_session
+from typing import Dict, Union
```
Comment (jdroenner) [inline geoengine/datasets.py:6]: is Union unused?

File: geoengine/datasets.py [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+class OgrOnError(Enum):
+    ignore = "ignore"
+    abort = "abort"
+
+
+def pandas_dtype_to_column_type(dtype: dtype) -> str:
+    type_map = {
+        'object': 'text',
+        'float64': 'float',
```
Comment (jdroenner) [inline geoengine/datasets.py:244]: what about the other float types?

File: geoengine/datasets.py [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+        'int64': 'int'
```
Comment (jdroenner) [inline geoengine/datasets.py:245]: what about other int types?

**Review by jdroenner — CHANGES_REQUESTED** [review]
please add time to the elements in the example / test

**Pull request merged**